frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -139,9 +139,13 @@ class Slider extends React.PureComponent<Props, State> {
       .style as React.CSSProperties
 
     return (
-      <div style={sliderOverrides.TickBar.style}>
-        <div style={tickBarItemStyle}>{sprintf(format, min)}</div>
-        <div style={tickBarItemStyle}>{sprintf(format, max)}</div>
+      <div className="sliderTickBar" style={sliderOverrides.TickBar.style}>
+        <div className="tickBarMin" style={tickBarItemStyle}>
+          {sprintf(format, min)}
+        </div>
+        <div className="tickBarMax" style={tickBarItemStyle}>
+          {sprintf(format, max)}
+        </div>
```

Comment:
This is an addition. Is it related to this test?

Reply:
I'm adding classNames to this to find it easier